server/src/main/java/org/candlepin/resource/OwnerResource.java

```diff
@@ -1213,58 +1091,29 @@ public void deleteOwner(@PathParam("owner_key") String ownerKey,
         return entitlementDTOs;
     }
 
-    /**
-     * Heals an Owner
-     *
-     * Starts an asynchronous healing for the given Owner. At the end of the
-     * process the idea is that all of the consumers in the owned by the Owner
-     * will be up to date.
-     *
-     * @param ownerKey id of the owner to be healed.
-     * @return a JobDetail object
-     * @httpcode 404
-     * @httpcode 202
-     */
-    @POST
-    @Produces(MediaType.APPLICATION_JSON)
-    @Consumes(MediaType.WILDCARD)
-    @Path("{owner_key}/entitlements")
-    @ApiOperation(notes = "Starts an asynchronous healing for the given Owner." +
-        " At the end of the process the idea is that all of the consumers " +
-        "in the owned by the Owner will be up to date.", value = "Heal owner")
-    @ApiResponses({ @ApiResponse(code = 404, message = "Owner not found") })
+    @Override
     public AsyncJobStatusDTO healEntire(
-        @ApiParam("ownerKey id of the owner to be healed.") @PathParam("owner_key") @Verify(Owner.class)
-        String ownerKey)
-        throws JobException {
+        @Verify(Owner.class) String ownerKey) {
 
         Owner owner = findOwnerByKey(ownerKey);
         JobConfig config = HealEntireOrgJob.createJobConfig().setOwner(owner).setEntitleDate(new Date());
 
-        AsyncJobStatus job = this.jobManager.queueJob(config);
+        AsyncJobStatus job = null;
+        try {
+            job = this.jobManager.queueJob(config);
+        }
+        catch (JobException e) {
+            e.printStackTrace();
```

Comment:
We should be doing what other endpoints do here, and use proper logging (`log.error()`) and then throwing an IseException

Reply:
Also, like the place that crog mentioned: we don't have to declare the job variable and do the translation outside the try block. It can just be:
```
        try {
            AsyncJobStatus job = this.jobManager.queueJob(config);
             return this.translator.translate(job, AsyncJobStatusDTO.class);
        }
        catch (JobException e) {
```
since there's no other work being done after the catch anyway

Reply:
Updated.